lib/rules/no-global-jquery.js

```diff
@@ -1,6 +1,15 @@
 'use strict';
 
 const { ReferenceTracker } = require('eslint-utils');
+const jqueryMethods = require('../utils/jquery-methods');
```

Comment:
Can you add one test case for the `no-global-jquery` rule to ensure it catches `jQuery.extend()` now?